api/spec/src/billing/invoice/cbc.tsp

```diff
@@ -0,0 +1,77 @@
+import "@typespec/http";
+import "@typespec/rest";
+import "@typespec/openapi3";
+
+using TypeSpec.Http;
+using TypeSpec.Rest;
+using TypeSpec.OpenAPI;
+
+namespace Invoice.CBC;
```

Comment:
Please explain what CBC stands for

Reply:
It's coming from [gobl](https://github.com/invopop/gobl/blob/5da023c15cd2ac2549e3d375d4614dd8aafa551c/cbc/cbc.go#L1-L4
).
```
// Package cbc provides a set of Common Basic Components.
//
// Name is take from the similar namespace used in UBL.
package cbc
```

Reply:
Ok, I am going to rename cbc and drift further away from the original schema.